### ISSUE TITLE
Design Intent - Accessibility Feedback - My Education Benefits, How to Apply Veteran Edu Experience, Application Recommender Tool

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#85618]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- I really like the direction this is going! Huge, huge fan of using what we already know to help Veterans make the right choices.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** Spend some time on the mobile view. Besides being a good design practice, modern smartphones have really solid screen readers and gesture controls built in. A lot of blind and low vision Veterans (especially younger ones) gravitate to their mobile device as their primary --- and sometimes only --- device. Make sure you have a plan for how everything is represented on mobile to ensure full feature support.
- [ ] **Must:** Echoing Erin's IA feedback, make sure you account for unhappy user paths. The one that stands out to me most is, "VA has something wrong about me, how do I fix it?" There's a cognitive load concern here: give users a clear path forward so they don't have to figure it out on their own.
- [ ] **Must:** As you refine your content, make sure your link text (eg. "Apply now") is clear about their purpose. A lot of screen reader users will navigate the page by tabbing through all of the links without their surrounding context, so we want to make sure it's clear what the link does from its text alone.
- [ ] **Should:** I think the "second scenario" info alert should have a heading as part of the alert. That makes the alert more discoverable for screen reader users (who often navigate by skipping heading-to-heading), and helps highlight the content visually in a way that doesn't rely on low-contrast blue-on-gray color.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone

**Comment by briandeconinck** [comment]
Apologies for the late feedback @alexander-ferzola!
